Formula/g/geni.rb

```diff
@@ -0,0 +1,19 @@
+class Geni < Formula
+  desc "Standalone database migration tool"
+  homepage "https://github.com/emilpriver/geni"
+  url "https://github.com/emilpriver/geni/archive/refs/tags/v1.0.6.tar.gz"
+  sha256 "89033304dc19bdab6c278d763ce06ae0b4f4d3175603e163c052f14ca5fe8140"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    ENV["DATABASE_URL"] = "sqlite3://test.sqlite"
+    system bin/"geni", "create"
+    assert_predicate testpath/"test.sqlite", :exist?, "failed to create test.sqlite"
```

Comment:
can we run a version assert as well? the rest lgtm

Reply:
Hey, ofc! on it :D

Reply:
Should be added now. I force pushed a new commit for the commit history :D